pages/Batches/Overview.md

```diff
@@ -0,0 +1,61 @@
+---
+menu: Batches
+name: Overview 
+route: /Batches/Overview 
+---
+
+# Batches
+
+The Batches specifications describe how Batches are announced, stored and queried.
+
+## File Format
+
+The storage file format is [Apache Parquet](https://github.com/apache/parquet-format).
+
+## Batch Announcements
+
+DSNP Messages are queued where they go directly into one or more DSNP Message Type queues. At some point when a queue is filled, the messages are dequeued and written to a Batch files and stored. The details of each created batch are sent to IAnnounce. IAnnounce emits one message per Batch file. Please see [Announce Message](/Messages/Announce) documentation for more details.
+
+
+## Batch file URLs
+
+The Batch file URL accepts different schemes to assist with knowing how to retrieve
+and query the file contents.  This allows adapters to be written to support different
+methods of storage and retrieval.
+
+For ordinary HTTP downloads, the SDK uses `http` or `https` as needed:
```

Comment:
I don't know if we have this elsewhere, but the whitepaper has a small note that we may reject `http` as an official option.

Reply:
good